tutorials/math/matrices_and_transforms.rst

```diff
@@ -151,13 +161,20 @@ it's used for 2D. The "X" axis is the element 0, "Y" axis is the element 1 and
 "Origin" is element 2. It's not divided in basis/origin for convenience, due to
 its simplicity.
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     var m = Transform2D()
     var x = m[0] # 'X'
     var y = m[1] # 'Y'
     var o = m[2] # 'Origin'
-    
+
+ .. code-tab:: csharp
+
+    var m = new Transform2D();
+    Vector2 x = m[0]; // 'X'
```

Comment:
Normally, it'd be better to use `var` but I agree that it's a good use of a type declaration here for an educational purpose.